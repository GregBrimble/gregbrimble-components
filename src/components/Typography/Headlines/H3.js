import React from "react"

import styled from "styled-components"
import { Headline } from "./shared"

const H3 = props => (
  <Headline
    as={styled.h3`
      font-size: 3rem;
      line-height: 3.125rem;
    `}
    {...props}
  />
)

export default H3
