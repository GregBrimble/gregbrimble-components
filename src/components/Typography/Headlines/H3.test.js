import React from 'react';
import H3 from './H3';
import renderer from 'react-test-renderer';

describe('H3', () => {
    it('renders properly', () => {
        const tree = renderer
            .create(<H3>Headline 1</H3>)
            .toJSON();
        expect(tree).toMatchSnapshot();
    });
});
