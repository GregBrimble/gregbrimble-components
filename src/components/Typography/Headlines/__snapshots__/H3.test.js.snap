// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`H3 renders properly 1`] = `
<h3
  className="sc-bdVaJa hEOvXI sc-htpNat dzkfLR"
>
  Headline 1
</h3>
`;
